src/NovaTable.vue

```diff
@@ -359,6 +364,12 @@ export default {
         },
     },
     methods: {
+        getRowClass(item) {
+            if (!this.rowClassCallback) {
+                return;
```

Comment:
This should return '', since I'm pretty sure `return;` would end up applying a class called 'undefined'.

It's also good practice to return the same type all the time (though we should break that rule when it makes sense).

Reply:
I'd like you to change `rowClassCallback` to just `rowClass`. Then I want it to be possible for a user to make it a string or function. If it's a string, obviously, just return the string. If it's a function, do what you've done here.

I'd like you to change the `tableClass` prop to work the same way.